frontend/src/pages/AddUser.tsx

```diff
@@ -0,0 +1,271 @@
+import {useEffect, useMemo, useState} from "react";
+import {Button, Form} from "react-bootstrap";
+import Card from "../components/Card";
+
+import classes from './AddUser.module.css';
+import useValidateForm from "../hooks/useValidateForm";
+import ValidationError from "../components/ValidationError";
+
+const MIN_USERNAME_LENGTH = 4;
+const MIN_PASSWORD_LENGTH = 12;
+
+interface AddUserProps {
+    isEdit:       boolean,
+    usernameInit: string,
+    passwordInit: string,
+    nameInit:     string,
+    surnameInit:  string,
```

Comment:
Enako kot zgoraj

Reply:
To so sam propsi, se ne pošiljajo na BE taka imena